CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed the Python and R Packages views incorrectly stating the default package
   files were missing even when they were present (#2882, #2884)
 
+### Changed
+
+- R manifest package generation now uses only `renv.lock` by default. To use the
+  local `renv` library instead, set `[r].packages_from_library = true` in the
+  configuration.
```

Comment:
Oh very interesting that you added this as a setting on the configuration rather than in the extension. I really like that 💯

Reply:
is `[r].packages_from_library` the right syntax? Should it be `[r.packages_from_library]` I thought that's how toml described nested properties?

Reply:
Also, agreed with Jordan: it's nice that this can be changed item by item if needed and isn't a publisher-wide thing.

Reply:
`[r.packages_from_library]` indicates the section `[packages_from_library]` under `[r]` in TOML syntax, that could be confusing for a reader. So I use `[r].packages_from_library` to make clear that it's attribute `packages_from_library` inside `[r]`